format/spec.md

```diff
@@ -1089,6 +1118,7 @@ The types below are not currently valid for bucketing, and so are not hashed. Ho
 
 | Primitive type     | Hash specification                        | Test value                                 |
 |--------------------|-------------------------------------------|--------------------------------------------|
+| **`unknown`**      | `hashInt(0)`                              |                                            |
```

Comment:
Sorry I missed this on my prior reviews, but shouldn't unknown be treated as null and return null for any cases that require a hash (also was `bucket[N]` included to be defined for 'unknown' if not then I think the only other case is puffin files which are today always defined as the hash of the values)?

Reply:
Yes, you're right. I'll fix it.

Reply:
Done.